crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -201,10 +203,11 @@ pub struct MeshUniform {
     pub inverse_transpose_model_a: [Vec4; 2],
     pub inverse_transpose_model_b: f32,
     pub flags: u32,
+    pub skin_index: u32,
 }
 
-impl From<&MeshTransforms> for MeshUniform {
-    fn from(mesh_transforms: &MeshTransforms) -> Self {
+impl MeshUniform {
+    pub fn from(mesh_transforms: &MeshTransforms, skin_index: u32) -> Self {
```

Comment:
```suggestion
    pub fn new(mesh_transforms: &MeshTransforms, skin_index: u32) -> Self {
```

Reply:
Agree with this change, as `from` has a special meaning in Rust wrt the `From`-trait (I am aware that you're aware, but potential new contributors might not)